src/Tribe/Assets.php

```diff
@@ -109,6 +109,7 @@ public function admin_enqueue_scripts() {
 			[
 				'groups'       => 'event-tickets-admin',
 				'conditionals' => [ $this, 'should_enqueue_admin' ],
+				'priority'     => 15,
```

Comment:
@juanfra Suggests we look at dependency on the other scripts needed.

Reply:
@sc0ttkclark I have checked it again and the dependency of `event-admin.js` is coming from TEC. So, we are not able to make it a dependency as we need load `ticket.js` in other supported custom post types besides `tribe-events`.

Reply:
Very good follow up @rafsuntaskin!

Should we set it as a dependency *if TEC is activated*?

Reply:
Great work Rafsun! Right, something like we do here but checking if TEC is there or not: https://github.com/moderntribe/event-tickets/blob/022e81ab515399ab9f6881d2dce3ef65eee03142/src/Tribe/Assets.php#L17

Reply:
Yes, we can do something like that for checking if TEC is available and then add that as a dependency. But this will be added as a group dependency for the whole `event-tickets-admin` asset group.

I have created another PR in common that can allow us to set individual dependencies while registering group assets:
https://github.com/moderntribe/tribe-common/pull/1428

Let me know what you think on that.

Reply:
> But this will be added as a group dependency for the whole event-tickets-admin asset group.

How? If you do something like the following it also add it as a dependency for the entire group?
```
$tickets_js_deps = [ 'jquery-ui-datepicker', 'tribe-bumpdown', 'tribe-attrchange', 'tribe-moment', 'underscore', 'tribe-validation', 'event-tickets-admin-accordion-js', 'tribe-timepicker' ];

// If TEC is active we add the dependency.
if ( class_exists( 'Tribe__Events__Main' ) ) { 
	$tickets_js_deps[] = 'tribe-events-admin';
}

$assets = [
	[ 'event-tickets-admin-css', 'tickets-admin.css', [ 'tribe-validation-style', 'tribe-jquery-timepicker-css', 'tribe-common-admin' ] ],
	[ 'event-tickets-admin-refresh-css', 'tickets-refresh.css', [ 'event-tickets-admin-css', 'tribe-common-admin' ] ],
	[ 'event-tickets-admin-tables-css', 'tickets-tables.css', [ 'event-tickets-admin-css' ] ],
	[ 'event-tickets-attendees-list-js', 'attendees-list.js', [ 'jquery' ] ],
	[ 'event-tickets-admin-accordion-js', 'accordion.js', [] ],
	[ 'event-tickets-admin-accordion-css', 'accordion.css', [] ],
	[ 'event-tickets-admin-js', 'tickets.js', $tickets_js_deps ],
];```

I'm not sure about touching the assets library of common because a lot of things rely on that. Have you tried making something similar to what I just shared above?